modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/engine/genesis.kt

```diff
@@ -34,13 +36,27 @@ open class AliceHas100XorAndPermissionToBurn : DefaultGenesis() {
     }
 }
 
+open class FewAssets : DefaultGenesis() {
```

Comment:
Maybe we will rename `FewAssets` to `MultipleAssets`? I believe it is more appropriate name